apollo-federation/src/sources/connect/json_selection/parameter_extraction.rs

```diff
@@ -0,0 +1,102 @@
+use std::collections::HashSet;
+
+use super::JSONSelection;
+use super::NamedSelection;
+use super::PathSelection;
+use super::SubSelection;
+
+/// A representation of a static parameter.
```

Comment:
maybe document the relationship between these parameters and the ones in url_templates? they're similar but not the same right?

Reply:
Sounds good! I have them separate so as to not tightly couple JSONSelection and the connector specific logic around parameters. I'll add more docs to make this more clear.